packages/ariakit/src/group/__examples__/group/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { Button, Group } from "ariakit";
```

Comment:
Oh, forgot about this one.
```suggestion
import { Button } from "ariakit/button";
import { Group } from "ariakit/group";
```